### PR TITLE
fix(occ): Add `user:auth-tokens` to command listing

### PR DESCRIPTION
**Pull request by joshtrichards**
These were implemented in nextcloud/server#40026 but never documented.

No usage examples, but at least the commands are now listed.

### ☑️ Resolves

* Fixes nextcloud/server#30918
* https://github.com/nextcloud/server/pull/40026#issuecomment-2011755402
* https://github.com/nextcloud/server/issues/44170#issuecomment-1998305551
* nextcloud/server#29001

### CODE REVIEW

File: admin_manual/occ_command.rst [hunk 1]
```diff
@@ -1362,16 +1362,21 @@ The ``--dry-run`` option can be used to simulate the restore without actually re
...
+The ``user`` commands create and remove users, reset passwords, manage authentication tokens / sessions, display a simple
 report showing how many users you have, and when a user was last logged in::
 
  user
   user:add                            adds a user
-  user:add-app-password               adds a app password named "cli"
+  user:add-app-password               adds a app password named "cli" (deprecated: alias for user:auth-tokens:add)
+  user:auth-tokens:add                Add app password for the named account
+  user:auth-tokens:delete             Deletes an authentication token
+  user:auth-tokens:list               List authentication tokens of an user
```
Comment (solracsf) [inline admin_manual/occ_command.rst:1373]: Just to be rephrased similarly to options below
```suggestion
  user:auth-tokens:list               List specified user authentication tokens

Comment (nickvergessen) [reply admin_manual/occ_command.rst:1373]:
```
occ user:auth-tokens:list   --help
Description:
  List authentication tokens of an user

```

If this should be reworded, it should be reworded in server first and then simply be aligned here

Comment (joshtrichards) [reply admin_manual/occ_command.rst:1373]: Yeah more polish would certainly be nice on the embedded command descriptions. I'd like to go through most of them one day, but that's a bigger job. I don't have the time or patience for it right now, particularly relative to more pressing priorities.

**Pull request merged**